com.unity.shadergraph/Editor/Data/Util/SpaceTransformUtil.cs

```diff
@@ -371,6 +375,17 @@ public static void AbsoluteWorldToWorld(SpaceTransform xform, string inputValue,
             }
         };
 
+        internal static IEnumerable<NeededTransform> ComputeTransformRequirement(SpaceTransform xform)
+        {
+            //Implicit always needed transform
+            yield return new NeededTransform(xform.from.ToNeededCoordinateSpace(), xform.to.ToNeededCoordinateSpace());
```

Comment:
I think technically we only need this transform if it's not Identity or ViaWorld.

Reply:
I actually moved this code from `public NeededTransform[] RequiresTransform(ShaderStageCapability stageCapability)`.
When xform.from is equals to xform.to, it won't generates any needtransform, no need for an extra test.
However, you are right, ViaWorld will always does "From A to World, from world to B", thus "From A to B" doesn't seems needed.
Let me double check ⏳

Reply:
Fix at 65a242ca48740b12866a86e87fbae5ca0f8b5873